timeplus/client.go

```diff
@@ -293,3 +314,110 @@ func (s *TimeplusClient) QueryStream(sql string) (rxgo.Observable, *QueryInfo, e
 	}()
 	return resultStream, &queryResult, nil
 }
+
+func (s *TimeplusClient) QueryV2Stream(sql string) (rxgo.Observable, error) {
+	query := Query{
+		SQL:         sql,
+		Name:        "",
+		Description: "",
+		Tags:        []string{},
+	}
+
+	createQueryUrl := fmt.Sprintf("%s/queries", s.baseUrlV2())
```

Comment:
I use https://github.com/r3labs/sse in our functional test. However, That library doesn't support "POST" version of SSE so I manually copied and pasted the code into our own repo: https://github.com/timeplus-io/neutron/blob/develop/tests/functional/functional_suite_test.go#L1227

I think we can contribute the POST version to them. It should be a minor change

Reply:
Yes, I did check that lib, we could consider use that lib in the future when we want to publish SDK.